### ISSUE TITLE
No labels in ./.../unlabel.cache

### ISSUE THREAD

**Issue by Dong-tranction** [AlibabaResearch/efficientteacher#41]
您好，我在实际半监督训练的时候一直报没有找到target的标签，target本来不就应该是无标签数据吗？为什么datasets_ssod.py加载数据的时候还要加载target的标签呢?
<img width="561" alt="屏幕截图 2023-03-22 170831" src="https://user-images.githubusercontent.com/71391484/226855862-5dda6a41-c1a0-4639-bdb4-9495f4b92ef6.png">

**Comment by BowieHsu** [comment]
@Dong-tranction 您好，请把cfg里面的ssod_hyp下的with_gt改为False，这个参数是为在做COCO实验时计算生成的伪标签和真实的标签之间的误差，用来调试训练代码策略。

**Comment by Dong-tranction** [comment]
> @Dong-tranction 您好，请把cfg里面的ssod_hyp下的with_gt改为False，这个参数是为在做COCO实验时计算生成的伪标签和真实的标签之间的误差，用来调试训练代码策略。

您好，我想请问下在readme里面提到的那个yaml文件是在具体什么位置，位置和名称是否已经改变了，我并没有找到。而且因为工程目录下yaml文件很多，我并不清楚具体应该使用那一个做为基础？

**Comment by Dong-tranction** [comment]
> @Dong-tranction 您好，请把cfg里面的ssod_hyp下的with_gt改为False，这个参数是为在做COCO实验时计算生成的伪标签和真实的标签之间的误差，用来调试训练代码策略。

我还想请教一个问题，我将训练和验证的img_size由640改为了【256， 384】，相应位置也作出了修改，在val.py中可以正常推理，但当我运行train.py时它会报以下错误，原因是有标签图片和无标签图片的tensor size无法torch.cat，矩阵的第三维和第四维交换了，请问是哪里还没有改过来么？
<img width="562" alt="屏幕截图 2023-03-23 185514" src="https://user-images.githubusercontent.com/71391484/227182231-1133e40d-52b2-48bf-af7c-1fe218ca3433.png">
